pkg/cluster/tls.go

```diff
@@ -60,7 +64,21 @@ func (m *manager) createCertificates(ctx context.Context) error {
 		m.log.Printf("waiting for certificate %s", c.certificateName)
 		err = m.env.ClusterKeyvault().WaitForCertificateOperation(ctx, c.certificateName)
 		if err != nil {
-			return err
+			m.log.Errorf("error when waiting for certificate %s: %s", c.certificateName, err.Error())
```

Comment:
I would be reluctant to add Cloud Errors before we understand timeout errors. In this case this might expose dirty backend errors to the users. And we are not in the control of the error content so it might leak something. 
I would like first to get more logging so we can evaluate production failures and understand what is happening. 

And once we know this we can return error to the user. Now this bit too verbose.